.gitignore

```diff
@@ -1,3 +1,4 @@
+.vscode/
```

Comment:
Hi :) This doesn't belong here. We don't add editor specific excludes to out .gitignore, because we cannot support all of the hundreds editors and other tools out there. Such excludes should go to the users global ~/.gitignore.

Can you remove it? Otherwise it's fine and I'm going to merge it afterwards.

Reply:
Okay, got it.